README.md

```diff
@@ -202,23 +208,39 @@ Returns the transactions collection by the lock or type script.
 #### Parameters
 
     search_key:
-        script - Script
+        script - Script, supports prefix search when group_by_transaction is false
         scrip_type - enum, lock | type
         filter - filter cells by following conditions, all conditions are optional
             script: if search script type is lock, filter cells by type script, and vice versa
             block_range: [u64; 2], filter cells by block number range, [inclusive, exclusive]
+        group_by_transaction - bool, optional default is `false`, if group_by_transaction is set to true, the returning objects will be grouped by the  x hash
     order: enum, asc | desc
     limit: result size limit
     after_cursor: pagination parameter, optional
 
 
 #### Returns
 
-    objects - transactions
+    objects - enum, ungrouped TxWithCell | grouped TxWithCells
+        TxWithCell:
+            tx_hash: transaction hash,
+            block_number: the number of the transaction committed in the block
+            tx_index: the position index of the transaction committed in the block
+            io_type: enum, input | output
+            io_index: the position index of the cell in the transaction inputs or outputs
+
+        TxWithCells:
+            tx_hash: transaction hash,
+            block_number: the number of the transaction committed in the block
+            tx_index: the position index of the transaction committed in the block
+            cells: Array [[io_type, io_index]]
+
     last_cursor - pagination parameter
 
 #### Examples
 
+get transactions by lock script
```

Comment:
There's no example for the return result. It's hard to tell how the enum looks like as JSON.

Reply:
All rpc methods currently have no examples of returning results, let me fix it in a new PR.